### ISSUE TITLE
I pressed some key combination and I'm seeing strange symbols instead of icons

### ISSUE THREAD

**Issue by blacklightpy** [Aylur/dotfiles#129]
I was in a dark room trying to press some shortcut and then some of my bar icons turned into strange characters.
![image](https://github.com/Aylur/dotfiles/assets/28739146/07184b8e-9057-4565-aba8-5cd2d4295311)
![image](https://github.com/Aylur/dotfiles/assets/28739146/18c0c884-b816-460c-be2d-621223a82b37)

**Comment by Balssh** [comment]
You need a nerdfont.

**Comment by blacklightpy** [comment]
@Balssh but I didn't have a nerdfont before and it was working..

**Comment by blacklightpy** [comment]
@Balssh Thank you, I installed `nerd-fonts-symbols-ttf` and it started working again. I had removed it because I thought I didn't really use it and `nerd-fonts` would take 4.7 GB with `nerd-fonts-otf` and `nerd-fonts-ttf`, but the symbols alone would only take 3.7 MB.

The reason I didn't notice it breaking anything was because I hadn't restarted ags until that point after uninstalling `nerd-fonts`.
